WordPress/gutenberg
# Apply elevation scale to: Modal, Popover, and Snackbar components

Following up on https://github.com/WordPress/gutenberg/pull/64108 to apply variables from the elevation scale to `Modal`, `Popover`, and `Snackbar` components. 

This change eliminates any instances of the deprecated `$shadow-popover` or `$shadow-modal` variables in the components package.

## Testing Instructions
1. `npm run storybook:dev`
2. Check the Modal, Popover, and Snackbar components
3. There should be no visual change
